client-src/elements/chromedash-guide-editall-page.js

```diff
@@ -135,9 +135,9 @@ export class ChromedashGuideEditallPage extends LitElement {
     return html`
       <form name="feature_form" method="POST" action="/guide/editall/${this.featureId}">
         <input type="hidden" name="token">
-        <input type="hidden" name="form_fields" value=${this.getFormFields()}>
+        <input type="hidden" name="form_fields" value=${this.getFormFields(formattedFeature.feature_type)}>
```

Comment:
You did not add a parameter to getFormFields().

Reply:
I see, sorry about that. I should have taken a cursory glance here to notice this, and it pertains to the hidden fields so it didn't seem to be non-functional. I've updated it with the change.